stake-pool/js/package.json

```diff
@@ -26,17 +26,21 @@
   "publishConfig": {
     "access": "public"
   },
-  "browser": "dist.browser/index.js",
-  "module": "dist.browser/index.esm.js",
-  "main": "dist/index.js",
+  "browser": {
+    "./dist/index.cjs.js": "./dist/index.browser.esm.js",
+    "./dist/index.esm.js": "./dist/index.browser.esm.js"
+  },
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
   "types": "dist/index.d.ts",
   "browserslist": [
     "defaults",
-    "not IE 11"
+    "not IE 11",
+    "maintained node versions"
   ],
   "files": [
     "/dist",
-    "/dist.browser"
+    "/src"
```

Comment:
probably just need `dist` now and not this, right?

Reply:
if we check for example https://github.com/solana-labs/solana-web3.js/blob/master/package.json , then we see that "src" is also included. I think that its better to include

Reply:
ah all right, fine by me!